javascripts/carlot.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+var CarLot = (function(){
+	var inventory = [];
+	return {
+		loadInventory: function (callback) {
+			var loader = new XMLHttpRequest();
+
+			loader.addEventListener("load", function () {
+				inventory = JSON.parse(this.responseText).cars;
+
+				return callback(inventory);
```

Comment:
you mean, I could just say "  callback(inventory);", right?  Yeah I realize that now